README.md

```diff
@@ -1,19 +1,307 @@
+| `Stack`  | Guardian stack   | `membership`                    |
+| `Stage`  | Environment      | `CODE` / `PROD`                 |
+
+### 🪣 S3 Buckets
+
+- **Content Bucket:** `manage-help-content` (configurable via CDK context)
+- **Deployment Bucket:** `membership-dist` (configurable via CDK context)
+
+### 🔐 IAM Permissions
+
+The Lambda functions have permissions to:
+
+- 📖 **Read** from deployment bucket
+- 📝 **Read/Write/Delete** from content bucket
+- 📋 **List** bucket contents
+
+## 📊 Monitoring & Alerts
+
+### 🚨 CloudWatch Alarms (PROD only)
+
+| Alarm                    | Threshold      | Action    |
+|--------------------------|----------------|-----------|
+| **Publisher 4xx Errors** | ≥ 5 errors/min | SNS Alert |
+| **Publisher 5xx Errors** | ≥ 5 errors/min | SNS Alert |
+| **Takedown 4xx Errors**  | ≥ 5 errors/min | SNS Alert |
+| **Takedown 5xx Errors**  | ≥ 5 errors/min | SNS Alert |
+
+### 📈 Metrics Dashboard
+
+Monitor your APIs through:
+
+- 📊 **CloudWatch Dashboards**
+- 🔍 **API Gateway metrics**
+- ⚡ **Lambda function metrics**
+- 🪣 **S3 access patterns**
+
+## 🤝 Contributing
+
+### 🔄 Development Workflow
+
+1. **🌿 Create feature branch** from `main`
+2. **💻 Make changes** with proper testing
+3. **✅ Run checks:** `pnpm package`
+4. **📝 Create Pull Request**
+5. **🤖 CI/CD runs** automatically
+6. **👥 Code review** by team
+7. **🚀 Merge** triggers deployment
+
+### 📝 Code Standards
+
+- **TypeScript** for CDK infrastructure
+- **Scala** for Lambda functions
+- **ESLint + Prettier** for code formatting
+- **Jest** for unit testing
+- **Guardian CDK** patterns and constructs
+
+## 🆘 Troubleshooting
+
+### 🔍 Common Issues
+
+| Issue                   | Solution                                        |
+|-------------------------|-------------------------------------------------|
+| **CDK synthesis fails** | Run `pnpm type-check` to find TypeScript errors |
+| **Tests failing**       | Check `pnpm test` output for specific failures  |
+| **Deployment errors**   | Check Riff-Raff logs and CloudWatch             |
+| **API errors**          | Monitor CloudWatch alarms and Lambda logs       |
+
+### 📞 Support
+
+- **📚 Documentation:** [RIFF-RAFF.md](./RIFF-RAFF.md)
+- **🐛 Issues:** GitHub Issues
+- **💬 Team Chat:** Guardian Slack channels
+- **📖 Guardian CDK:** [CDK Documentation](https://github.com/guardian/cdk)
+
+## 📚 Additional Resources
+
+- **🏗️ [Guardian CDK](https://github.com/guardian/cdk)** - Infrastructure patterns
+- **🚀 [Riff-Raff Documentation](./RIFF-RAFF.md)** - Deployment guide
+- **☁️ [AWS CDK](https://docs.aws.amazon.com/cdk/)** - Official CDK docs
+- **📋 [Salesforce Knowledge](https://gnmtouchpoint.lightning.force.com/)** - Content source
+- **🌐 [MMA Help Centre](https://manage.theguardian.com/help-centre)** - Published content
+
+---
+
+<div align="center">
+
+**🎉 Successfully migrated to Guardian CDK!**
+
+*Built with ❤️ by the Guardian Membership team*
+
+</div>
```

Comment:
Cute but I'd drop this bit 😊 "migrated to CDK" is only relevant for the PR, and we're the Supporter Revenue stream (membership was a legacy product back near the start of the Guardian's digital reader revenue journey, but a lot of our tech stack still references it)